scripts/rescript_bsb.js

```diff
@@ -6,47 +6,47 @@ var os = require("os");
 const child_process = require("child_process");
 const rescript_exe = require("./bin_path").rescript_exe;
 
-/**
- * @typedef {Object} ProjectFiles
- * @property {Array<string>} dirs
- * @property {Array<string>} generated
- */
-
-/**
- * @typedef {Object} WatcherRef
- * @property {string} dir
- * @property {fs.FSWatcher} watcher
- */
```

Comment:
I think the type annotations were used somewhere below

Reply:
I guessed so, but never touched it. Can I just delete them?

Reply:
I think it's better to keep it, so the type checking works better.

Reply:
I will try to find the correct location for it. If it is dangling, it has no meaning and should be deleted.

Reply:
oh it was there for type definition. rollback